.github/workflows/playwright.yml

```
@@ -0,0 +1,34 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v6
+
+      - uses: ./.github/actions/setup-bun
+        with: { install-node: "true" }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)